content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -356,6 +356,58 @@ prisma init --datasource-provider sqlite
 
 The command output contains helpful information on how to use the generated files and begin using Prisma with your project.
 
+#### Run `prisma init --preview-feature`
+
+<CodeWithResult outputResultText="Prisma schema" expanded={true}>
+
+<cmd>```terminal prisma init --preview-feature multiSchema```</cmd>
```

Comment:
This is broken and not displayed @ruheni 
<img width="538" alt="image" src="https://github.com/prisma/docs/assets/183673/66d3334d-5b72-4bad-8124-fd9eadd9a759">

Reply:
Created a PR here: https://github.com/prisma/docs/pull/5410